platformio/src/locales/locale_fr_FR.inc

```diff
@@ -0,0 +1,289 @@
+//   see English alerts in territories where English is uncommon. To satisify
+//   this, users should uncomment the DISABLE_ALERTS macro in config.h.
+//
+// Here are a few examples, uncomment the array for your region (or create your
+// own).
+// const std::vector<String> ALERT_URGENCY = {"statement", "watch", "advisory", "warning", "emergency"}; // US National Weather Service
+const std::vector<String> ALERT_URGENCY = {"yellow", "amber", "red"};                 // United Kingdom's national weather service (MET Office)
+// const std::vector<String> ALERT_URGENCY = {"minor", "moderate", "severe", "extreme"}; // METEO
+// const std::vector<String> ALERT_URGENCY = {}; // Disable urgency interpretation (algorithm will fallback to only prefer the most recently issued alerts)
+
+// ALERT TERMINOLOGY
+// Weather terminology associated with each alert icon
+const std::vector<String> TERM_SMOG =
+    {"brouillard"};
+const std::vector<String> TERM_SMOKE =
+    {"fumée"};
+const std::vector<String> TERM_FOG =
+    {"brouillard", "haar"};
+const std::vector<String> TERM_METEOR =
+    {"meteor", "asteroid"};
+const std::vector<String> TERM_NUCLEAR =
+    {"nuclear", "radiation ionisante"};
+const std::vector<String> TERM_BIOHAZARD =
+    {"biohazard", "risque biologique"};
+const std::vector<String> TERM_EARTHQUAKE =
+    {"tremblement de terre"};
+const std::vector<String> TERM_TSUNAMI =
+    {"tsunami"};
+const std::vector<String> TERM_FIRE =
+    {"feu", "drapeau rouge"};
+const std::vector<String> TERM_HEAT =
+    {"chaleur"};
+const std::vector<String> TERM_WINTER =
+    {"blizzard", "hiver", "glace", "neige", "grésil", "froid", "pluie verglaçante", 
+     "vent refroidissant", "gel", "givre", "grêle"};
+const std::vector<String> TERM_LIGHTNING =
+    {"orage", "cellule orageuse", "pulse storm", "ligne de grain", "supercellule",
+     "éclair"};
+const std::vector<String> TERM_SANDSTORM =
+    {"tempête de sable", "poussiere souflée", "tempete de poussiere"};
+const std::vector<String> TERM_FLOOD =
+    {"inondation", "onde de tempête", "seiche", "gonflement", "mer haute", "grande marée",
+     "raz-de-marée"};
+const std::vector<String> TERM_VOLCANO =
+    {"volcanic", "cendre", "volcan", "éruption"};
+const std::vector<String> TERM_AIR_QUALITY =
+    {"air", "stagnation", "pollution"};
+const std::vector<String> TERM_TORNADO =
+    {"tornade"};
+const std::vector<String> TERM_SMALL_CRAFT_ADVISORY =
+    {"petit navire", "avis de vent"};
+const std::vector<String> TERM_GALE_WARNING =
+    {"coup de vent"};
+const std::vector<String> TERM_STORM_WARNING =
+    {"storm warning"};
+const std::vector<String> TERM_HURRICANE_WARNING =
+    {"vent de type ouragan", "vent extreme", "grand vent"};
+const std::vector<String> TERM_HURRICANE =
+    {"ouragan", "tempete tropicale", "typhoon", "cyclone"};
+const std::vector<String> TERM_DUST =
+    {"poussière", "sable"};
+const std::vector<String> TERM_STRONG_WIND =
+    {"vent"};

```

Comment:
> //   OpenWeatherMap returns alerts in English regardless of the OWM LANGUAGE
> //   option or territory. For this reason it is preferred that you do not
> //   translate text related to Alerts.

This text is not shown to the user, just used to display the correct icon. As mentioned above, OpenWeatherMap only provides alerts in English. 

```suggestion
const std::vector<String> TERM_SMOG =
    {"smog"};
const std::vector<String> TERM_SMOKE =
    {"smoke"};
const std::vector<String> TERM_FOG =
    {"fog", "haar"};
const std::vector<String> TERM_METEOR =
    {"meteor", "asteroid"};
const std::vector<String> TERM_NUCLEAR =
    {"nuclear", "ionizing radiation"};
const std::vector<String> TERM_BIOHAZARD =
    {"biohazard", "biological hazard"};
const std::vector<String> TERM_EARTHQUAKE =
    {"earthquake"};
const std::vector<String> TERM_TSUNAMI =
    {"tsunami"};
const std::vector<String> TERM_FIRE =
    {"fire", "red flag"};
const std::vector<String> TERM_HEAT =
    {"heat"};
const std::vector<String> TERM_WINTER =
    {"blizzard", "winter", "ice", "snow", "sleet", "cold", "freezing rain", 
     "wind chill", "freeze", "frost", "hail"};
const std::vector<String> TERM_LIGHTNING =
    {"thunderstorm", "storm cell", "pulse storm", "squall line", "supercell",
     "lightning"};
const std::vector<String> TERM_SANDSTORM =
    {"sandstorm", "blowing dust", "dust storm"};
const std::vector<String> TERM_FLOOD =
    {"flood", "storm surge", "seiche", "swell", "high seas", "high tides",
     "tidal surge"};
const std::vector<String> TERM_VOLCANO =
    {"volcanic", "ash", "volcano", "eruption"};
const std::vector<String> TERM_AIR_QUALITY =
    {"air", "stagnation", "pollution"};
const std::vector<String> TERM_TORNADO =
    {"tornado"};
const std::vector<String> TERM_SMALL_CRAFT_ADVISORY =
    {"small craft", "wind advisory"};
const std::vector<String> TERM_GALE_WARNING =
    {"gale"};
const std::vector<String> TERM_STORM_WARNING =
    {"storm warning"};
const std::vector<String> TERM_HURRICANE_WARNING =
    {"hurricane force wind", "extreme wind", "high wind"};
const std::vector<String> TERM_HURRICANE =
    {"hurricane", "tropical storm", "typhoon", "cyclone"};
const std::vector<String> TERM_DUST =
    {"dust", "sand"};
const std::vector<String> TERM_STRONG_WIND =
    {"wind"};
```